ch8/grafana-docker-compose.yaml

```
@@ -0,0 +1,15 @@
+version: "3"
+
+services:
+  grafana-dashboard:
+    image: grafana/grafana
+    ports:
+     - 3000:3000
+    environment:
+      GF_SECURITY_ADMIN_USER: dashboarduser
+      GF_SECURITY_ADMIN_PASSWORD: dashboardpw
```

```suggestion
      GF_SECURITY_ADMIN_PASSWORD: dashboardpassword
```
통일성을 위해 password 로 바꾸면 어떨까요?

---

Done.